### DISCUSSION TITLE
questions about scale set API endpoint in ARC

### DISCUSSION THREAD

**Discussion by yuwei1chen** [actions/actions-runner-controller#3255] [Questions]
Hi, I am studying ARC. I see that ARC is using below API endpoint to manage scale sets on github.
https://github.com/actions/actions-runner-controller/blob/master/github/actions/client.go#L26
May I ask what this API endpoint is? I couldn't find what "ActionsServiceURL" in code is. I also could not find this API endpoint in Github's rest docs.

**Comment by Shegox** [comment]
Related: https://github.com/actions/actions-runner-controller/discussions/3289
